core/editor/config-providers/editor-v2-config-provider.php

```diff
@@ -72,6 +72,7 @@ public function get_script_handles_to_enqueue() {
 				return $script_asset['handle'];
 			} )
 			->push( 'elementor-editor-loader-v2' )
+			->push( 'elementor-responsive-bar' )
```

Comment:
Why it should be loaded?

Reply:
I wanted this PR to have a working BPs, and the next one removes it. This PR just extracts the responsive-bar and it keeps working the same.